test/MsBuildScript.Tests/runtests.target

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="RunMsBuildScriptTests" 
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="../../dir.props" />
```

Comment:
I think this test looks good if there is some risk that the dependency files don't exist in blob storage.

I would adjust the naming though.  I would use names that describe more specifically what it does, instead of calling these generic "MSBuild script tests".  So something like `CheckForBlobStorageDependencies` instead of `TestsForBuildItself`.  Also I'm not sure if there's much reason to follow the pattern of having a separate project file which is invoked via the `DotNetMSBuild` task.  I would probably remove that indirection.